src/csync/csync_exclude.cpp

```diff
@@ -330,8 +323,16 @@ bool ExcludedFiles::reloadExcludeFiles()
     bool success = true;
     const auto keys = _excludeFiles.keys();
     for (const auto& basePath : keys) {
-        for (const auto& file : _excludeFiles.value(basePath)) {
-            success = loadExcludeFile(basePath, file);
+        for (const auto &excludeFile : _excludeFiles.value(basePath)) {
+            QFile file(excludeFile);
+            if (file.open(QIODevice::ReadOnly)) {
+                if (file.exists()) {
```

Comment:
should we not check `if (file.exists())` before trying to open the file ?
if I understand correctly this file should be always synced even with virtual files